tokens/src/components/Form/transition.json

```diff
@@ -1,9 +1,14 @@
 {
   "transition": {
     "form": {
-      "input": { "value": "border-color .15s ease-in-out, box-shadow .15s ease-in-out", "source": "$input-transition" },
+      "input": {
+        "value": "border-color .15s ease-in-out, box-shadow .15s ease-in-out",
+        "type": "transition",
```

Comment:
[consideration] Curious for your thoughts on whether we should define transitions as a [composite token](https://design-tokens.github.io/community-group/format/#transition)? Primary rationale being that having the separate parts helps define the style properties in a more platform agnostic way than this CSS implementation. It would likely require a custom transform, though.